packages/lib-panoptes-js/src/resources/projects/rest.js

```diff
@@ -0,0 +1,41 @@
+const panoptes = require('../../panoptes')
+const { handleError, endpoint } = require('./helpers')
+
+function create (params) {
+  const newProjectData = (params && params.data) ? params.data : {}
+
+  const allProjectData = Object.assign({
+    private: true
+  }, newProjectData)
+
+  return panoptes.post(endpoint, allProjectData)
+}
+
+function get (params) {
+  const queryParams = (params && params.query) ? params.query : {}
+  const projectId = (params && params.id) ? params.id : ''
+
+  if (!projectId) return panoptes.get(endpoint, queryParams)
+  if (projectId && typeof projectId !== 'string') return handleError('Projects: Get request id must be a string.')
+
```

Comment:
Should this be `handleError(new Error('Projects: Get request id must be a string.')`?

Reply:
Thanks for the examples. I'll update this.